frontend/src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactChild } from 'react'
+import { IconType } from 'react-icons/lib'
+import {
+  Box,
+  forwardRef,
+  HStack,
+  Icon,
+  Text,
+  useMultiStyleConfig,
+  VStack,
+} from '@chakra-ui/react'
+
+export interface TileProps {
+  // The typing here is to satisfy the ts compiler
+  // because otherwise, it will complain about assigning null to the as prop
+  /**
+   * The icon that should be displayed on the tile
+   */
+  icon: IconType
+
+  /**
+   * The title of the tile
+   */
+  title: string
+  /**
+   * The subtitle of the tile
+   */
+  subtitle: string
+
+  /**
+   * The components to be displayed below the title/subtitle
+   */
+  children: ReactChild
+
+  /**
+   * The tag, if any, to be displayed alongside the title
+   */
+  tag?: JSX.Element
+}
+
+type TileVariant = 'complex' | 'simple'
+export const Tile = forwardRef<TileProps, 'div'>(
+  ({ tag, icon, title, subtitle, children }, ref) => {
+    const variant: TileVariant = children ? 'complex' : 'simple'
+    const styles = useMultiStyleConfig('Tile', { variant })
+
+    return (
+      // Ref passed into the component as a whole so that it can be focused
+      <Box sx={styles.container} ref={ref}>
+        <VStack spacing="1rem" alignItems="flex-start">
+          <HStack spacing="1rem">
+            <Icon sx={styles.icon} as={icon}></Icon>
+            {tag}
+          </HStack>
+          <Box>
+            <Text sx={styles.title} as="h4">
+              {title}
+            </Text>
+            <Text sx={styles.subtitle}>{subtitle}</Text>
+          </Box>
```

Comment:
Can export the Description, Title and Subtitle wrappers as a subcomponent so outside users can build their own tag with their own styling (if needed)

Reply:
could i clarify on this point - i actually decided _against_ doing this for the reasons stated in the PR write-up and i wanted to know when this is preferred (or is this all the time b/c we r building our own design system)

Reply:
> This is because the Tile component is more of a wrapper component

Precisely what you said in your PR description? So you expose `Tile.Tile`, `Tile.Title`, `Title.Description`, in addition to the main `Tile` component that uses all of those things precisely so users can modify any part of the component if they want instead of building another one, since all it is is a wrapper containing various styling

Reply:
gotcha - to confirm this: wrapper components should export their wrapped components because it helps people to style components if needed; but wrt this, we shouldn't export  a `Description` component directly right? 

primarily because this is a wrapper and doesn't contribute anything of value beyond the chakra tags so we should export the components that are different (i took this approach)